src/webrtc/congestion_control/cc.md

```diff
@@ -0,0 +1,164 @@
+
+The whole process is shown in the following figure.
+
+![GCC standard flow](./gcc_std_flow.png)
+
+One can use the fact, that in the decrease state estimation is set to 85& of the latest receiver rate and 
+set sender rate more aggressively which should result in faster convergence.
+
+![GCC overuse flow](./gcc_overuse_flow.png)
+
+## GCC & Simulcast
+
+Let's consider a scenario where the network link between the server and the receiver is quite poor.
+In such case, the sender will send high-quality video to the server but the receiver won't be able to receive it.
+Limiting media quality on the server side is challenging as sending video with a rate lower than the video
+bitrate will result in frequent video rebuffering (video freezes).
+What the server can do is request the sender to e.g. increase its compression level, decrease resolution or FPS,
+or reencode the video on its own.
+The former is problematic in sessions with more than 2 participants as there might be e.g. only one host that
+cannot handle high-quality video but the server will send all of the participants lower resolution.
+The latter requires a lot of resources and is hard to be achieved in real-time.
+This leads us to the simulcast where the sender sends multiple qualities in multiple FPS and the server decides
+which quality should be sent to which participant.
+There are still some bitrate levels so the server cannot lower its rate to any level but in most cases,
+there is always some quality that will fit into the receiver network condition.
+
+![simulcast overview](./simulcast.png)
+
+As stated before, bandwidth estimation depends on the amount of data that is sent therefore, if the server sends
+little data the estimation will be low.
+To recover from this situation, the server has to increase the amount of data it sends which leads us to a connection probing.
+
+## Connection Probing
+
+When the server wants to recover from sending a low resolution to some peer, it has to generate additional traffic
+to see if the network can handle more data.
+
+Unfortunately, the connection probing is not standardized so every SFU might do this in a different way.
+
+The questions we need to answer are:
+1. What data use for probing?
+1. How much to probe?
+1. How often to probe?
+
+### 1. What data use for probing
+
+The naive implementation would be to switch to the higher layer every `x` seconds and observe the network.
+This, however, may lead to periodic video rebuffering and lower Quality of Experience (QoE).
+
+Another solution is to generate traffic by sending RTP padding packets.
+In this case, the server has full control over the amount of data being sent and therefore, it can probe with 
+different aggressiveness.
+
+The problem with padding packets is that they contain only zeros (except the last byte) so we end up sending
+a lot of garbage data, useless for the client.
+Fortunately, there is a solution which is called RTX.
```

Comment:
Ideally, I think we should have a reference to another chapter about RTX here. However, for the time being... I wouldn't use RTX, it's just a MIME type of this payload type :stuck_out_tongue: 
```suggestion
Fortunately, we can also retransmit previously sent packets in order to probe the connection. For the purposes of this chapter, it's not important to know how the retransmissions really work, but you can read more about it in RFC 4588.
```

Reply:
IMO it's important to make it clear that RTX it is used for probing. Also, if we decide to use RTX for connection probing in the future, we will probably describe some nuances here too, similarly to the `Probing with padding packets`

Reply:
Yeah, but I would use the full name